default_security_group.tf

```diff
@@ -12,7 +12,7 @@ locals {
 
 resource "ibm_is_security_group_rule" "default_vpc_rule" {
   for_each  = local.security_group_rule_object
-  group     = ibm_is_vpc.vpc.default_security_group
+  group     = ibm_is_vpc.vpc[0].default_security_group
```

Comment:
Hmm, this means this code block will only occur if a new VPC is created. Won't we want it to happen on an existing VPC too if one is passed ? @toddgiguere @digvijay-ukirde ?